### PR TITLE
fix(storefront): BCTHEME-424 Alt text should include product name for ratings

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

Alt text is provided for 'Star rating' of the PLPs should include the product name along with the rating.

#### Tickets / Documentation

[Jira](https://jira.bigcommerce.com/browse/BCTHEME-424)

#### Screenshots (if appropriate)

Screenshot 2021-03-18 at 13 42 10

### CODE REVIEW

File: templates/components/products/ratings.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <span role="img"
-      aria-label="{{lang 'products.reviews.rating_aria_label' current_rating=rating max_rating=5}}"
+    {{#if product_name}}
+      aria-label="{{lang 'products.reviews.rating_aria_label' rating_target=rating_target current_rating=rating max_rating=5}}"
```
Comment (yurytut1993) [inline templates/components/products/ratings.html:3]: do we have cases when product name can be undefined?

**Comment by yurytut1993** [comment]
I believe we also should update rating for amp pages

File: templates/components/products/ratings.html (hunk 1 above)
Comment (BC-tymurbiedukhin) [reply templates/components/products/ratings.html]: deprecated var name, fixed

**Comment by BC-tymurbiedukhin** [comment]
> I believe we also should update rating for amp pages

Done

**Review by yurytut1993 — APPROVED** [review]

**Pull request merged**